### PR TITLE
Add more information to AUTHORS.md

### PR DESCRIPTION
**Pull request by PaulWessel**
Such as who contributed supplements.

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -28,7 +35,8 @@ Garry Vaughan, William Weibel, and many others, including their advice on how
 to make GMT portable to a wide range of platforms. John Lillibridge and Stephan
 Eickschen provided the original Examples (11) and (32), respectively; Hanno von
 Lom helped resolve early problems with DLL libraries for Win32; Lloyd Parkes
-enabled indexed color images in PostScript; Kurt Schwehr maintains the fink packages;
+enabled indexed color images in PostScript; Brent Wood developed the GMT/OGR format;
+Kurt Schwehr maintains the fink packages;
```
Comment (joa-quim) [inline AUTHORS.md]: Hmm, but we don't acknowledge the other package maintainers!

Comment (PaulWessel) [reply AUTHORS.md]: Feel free to expand on this file!

Comment (joa-quim) [reply AUTHORS.md]: Well, what I meant is that acknowledging package managers is a bit beyond the purpose of this list.

Comment (PaulWessel) [reply AUTHORS.md]: That is true, agree we can remove that.

**Comment by seisman** [comment]
Maybe also acknowledge the Steering Committee?

**Comment by PaulWessel** [comment]
No, they are not authors and we have a separate repo for oversight.  I removed package managers.

**Pull request merged**